### PR TITLE
Add revision 2

### PR DESCRIPTION
**Pull request by kleberbaum**
- Fix faulty scaling
 - Fix wrong font size/font
 - Add hover effect when hovering cards
 - Add hover effect from center
 - Add hamburger menu closing after clicking

### CODE REVIEW

**Comment by schettn** [comment]
- Handwritten font does not load
- Großhandel distributor images are not loading
- Dekoration /contentpage1 hero image not loading when there is a trailing slash (/contentpage1/)

**Comment by schettn** [comment]
```diff
diff --git a/package.json b/package.json
index 5700e1f..e27a944 100644
--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-icons": "^4.6.0",
+    "react-simple-image-viewer": "^1.2.2",
     "react-slick": "^0.29.0",
     "sharp": "^0.31.2",
     "slick-carousel": "^1.8.1"
diff --git a/src/components/Navigation/MobileNav/MobileNav.tsx b/src/components/Navigation/MobileNav/MobileNav.tsx
index c84dabd..7a70b8e 100644
--- a/src/components/Navigation/MobileNav/MobileNav.tsx
+++ b/src/components/Navigation/MobileNav/MobileNav.tsx
@@ -11,7 +11,7 @@ import {
 import { FC } from "react"
 import { AiOutlineShoppingCart, AiOutlineUser } from "react-icons/ai"
 import { IoCloseOutline, IoMenuOutline } from "react-icons/io5"
-import { Link } from "react-router-dom"
+import { Link } from "gatsby"
 import { navlinks } from "../../../constant/navLink"
 
 interface IMobileNavProps {}
```

File: src/components/Navigation/MobileNav/MobileNav.tsx [hunk 1]
```diff
@@ -1,76 +1,100 @@
...
-import { navlinks } from '../../../constant/navLink';
-import { FC } from 'react';
-import { AiOutlineShoppingCart, AiOutlineUser } from 'react-icons/ai';
-import { IoCloseOutline, IoMenuOutline } from 'react-icons/io5';
-import { Link } from 'gatsby';
+} from "@chakra-ui/react"
+import { FC } from "react"
+import { AiOutlineShoppingCart, AiOutlineUser } from "react-icons/ai"
+import { IoCloseOutline, IoMenuOutline } from "react-icons/io5"
+import { Link } from "react-router-dom"
```
Comment (kleberbaum) [inline src/components/Navigation/MobileNav/MobileNav.tsx:14]:
```suggestion
import { Link } from "gatsby"
```

File: package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
     "react-dom": "^18.1.0",
     "react-icons": "^4.6.0",
```
Comment (kleberbaum) [inline package.json:22]:
```suggestion
    "react-icons": "^4.6.0",
    "react-simple-image-viewer": "^1.2.2",
```

**Review by kleberbaum — COMMENTED** [review]
Should be ok to merge after a few changes.

**Pull request merged**